upgrade/1.0.1/Stage_2.md

```diff
@@ -8,6 +8,8 @@
 > ncn# unset CRAY_FORMAT
 >```
 
+1. In order for nodes to properly PXE boot, Border Gateway Protocol \(BGP\) must be healthy.  Before proceeding, check the status of BGP as described at [Check BGP Status and Reset Sessions](../../operations/network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md).
```

Comment:
Change "at Check BGP Status and Reset Sessions" to "in the Check BGP Status and Reset Sessions procedure"

Reply:
fixed